services/movie_session.py

```diff
@@ -0,0 +1,44 @@
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: int,
+        movie_id: id,

```

Comment:
The parameter `movie_id` should be of type `int` instead of `id`. Update the function signature to `movie_id: int`.